From 82c11604a518d7c5c6f318e9c2de9406792990ef Mon Sep 17 00:00:00 2001
From: Dom Cobley <popcornmix@gmail.com>
Date: Sun, 31 Oct 2021 11:47:59 +0000
Subject: [PATCH 1214/2201] bcm2835-v4l2-codec: Remove advertised support of
 VP8

The support for this format by firmware is very limited
and won't be faster than the arm.

Signed-off-by: Dom Cobley <popcornmix@gmail.com>
---
 .../staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c b/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
index e8452c82d088..351d693873ed 100644
--- a/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
+++ b/drivers/staging/vc04_services/bcm2835-codec/bcm2835-v4l2-codec.c
@@ -597,11 +597,6 @@ static const struct bcm2835_codec_fmt supported_formats[] = {
 		.depth			= 0,
 		.flags			= V4L2_FMT_FLAG_COMPRESSED,
 		.mmal_fmt		= MMAL_ENCODING_MP2V,
-	}, {
-		.fourcc			= V4L2_PIX_FMT_VP8,
-		.depth			= 0,
-		.flags			= V4L2_FMT_FLAG_COMPRESSED,
-		.mmal_fmt		= MMAL_ENCODING_VP8,
 	}, {
 		.fourcc			= V4L2_PIX_FMT_VC1_ANNEX_G,
 		.depth			= 0,
-- 
2.17.1

